styles/_shop-btn.scss

```diff
@@ -0,0 +1,68 @@
+.shop-item__btns {
+  display: flex;
+
+}
+
+.shop-btn {
+  display: flex;
+  justify-content: center;
+  width: 20%;
+  padding: 0.5rem;
+  border-top: $brand-border;
+  border-right: $brand-border;
+
+  &:last-of-type {
+    border-right: 0;
+  }
+}
+
+.dropdown {
+  &:hover {
+    position: relative;
+    background: $brand-primary;
+    color: $brand-white;
```

Comment:
`position: relative` probably shouldn't be a result of the hover.

The changes to `background` and `color` will should be part of any button, whether it is a drop down or not.